docs/release-notes.md

```diff
@@ -1,5 +1,7 @@
 # Release Notes
 
+* [Percona Distribution for MySQL 8.0.28 using Percona XtraDB Cluster](release-notes-pxc-v8.0.28.md)
```

Comment:
These links still miss the release dates - are you going to add these as a separate PR?

Reply:
No, I will update this one once the release dates are defined